bench/wtperf/wtperf.c

```diff
@@ -456,6 +486,79 @@ worker(void *arg)
 		goto err;
 	}
 
+	/* Setup for truncate */
+	if (thread->workload->truncate != 0) {
+
+		STAILQ_INIT(&truncate_stone_head);
```

Comment:
Once this moves into the CONFIG structure, the initializer belongs somewhere shared, rather than in a worker. Maybe in ```start_run```.